routes/userRoutes.js

```diff
@@ -88,6 +89,56 @@ async function validateFields(fields) {
   return true;
 }
 
+// Get users subscriptions
+router.get('/:id/subscriptions', async (req, res) => {
+  try {
+    const userId = req.params.id;
+
+    // Find the user by _id and select the 'subscriptions' field
+    const user = await User.findById(userId).select('subscriptions');
+
+    if (!user) {
+      return res.status(404).json({ message: 'User not found' });
```

Comment:
Should return error 204 as stated in microsoft API guidelines (error 204 can't return a response)

Reply:
<img width="980" alt="image" src="https://github.com/Educado-App/educado-backend/assets/92582321/5459c47d-70e2-4365-b381-b18e4fdcfd85">

Reply:
...?

Reply:
Jeg tror det kun gælder for DELETE requests. Jeg spurgte chatGPT for at dobbelttjekke:

Yes, using a 404 status code in this context is appropriate, assuming you are following RESTful principles for your microservices API. In your code, you're attempting to retrieve a user's subscriptions by their ID. If the user with the given ID doesn't exist, returning a 404 status code is a valid response.

HTTP status code 404 (Not Found) is used to indicate that the requested resource (in this case, the user) was not found on the server. It's a standard way to communicate to the client that the requested data doesn't exist.

So, your code is correctly using a 404 status code to indicate that the user was not found, and it provides a relevant error message in the JSON response ("User not found"). This is in line with RESTful API design best practices.